priam/src/main/java/com/netflix/priam/identity/InstanceIdentity.java

```diff
@@ -267,12 +267,6 @@ private void populateInstancesByAvailabilityZoneMultiMap() {
             seeds.add(instancesByAvailabilityZoneMultiMap.get(loc).get(0).getHostIP());
         }
 
-        // Remove this node from the seed list so Cassandra auto-bootstrap will kick in.  Unless this is the only node
-        // in the cluster.
-        if (seeds.size() > 1) {
-            seeds.remove(myInstance.getHostIP());
-        }
-
```

Comment:
Just documenting my reply here from our offline conversation:
So, if I am bootstrapping, and I can't connect to other seeds, then I want to crash and burn.. instead of thinking I am the only one in the cluster